.github/workflows/crawler.yml

```diff
@@ -69,13 +75,16 @@ jobs:
         run: |
           [ -d data ] && echo data/* | xargs git add
           git add -u data/
-
-          shopt -s nullglob # https://stackoverflow.com/a/31054582/902217
-          for yaml in index*.yaml; do
-            zip_name="${yaml%.*}.zip"
-            zip "${zip_name}" "${yaml}"
-            git add "${zip_name}" "${yaml}"
-          done
+          
+          if ${{ inputs.commit-indexes }}
```

Comment:
Maybe it will be easier to add this to condition in the next step which do actual commit

Something like

```
- if: github.event_name != 'pull_request' && steps.diff.outputs.skip_commit != 'true' && inputs.commit-indexes
```

Reply:
Oh i see, thanks for note. My commend doesn't make sense in this case, resolving